### PR TITLE
Create GitBooks site

### PR DESCRIPTION
**Pull request by shawndwells**
This PR establishes the initial GitBooks scaffolding and table of contents. The project's primary README is also updated for clarity.

* PR is built against my fork and displayed at https://crowdstrike.gitbook.io/cloud-aws/. Once this PR is merged, will cut over the https://crowdstrike.gitbook.io/cloud-aws/ to this upstream repo.
* Updated the wordlist by adding `deregister`, then sorted the whole file alphabetically.

### CODE REVIEW

**Comment by shawndwells** [comment]
Scaffolding to get https://crowdstrike.gitbook.io/cloud-aws/ in order.

Right now https://crowdstrike.gitbook.io/cloud-aws/ builds from my local fork.

**Comment by shawndwells** [comment]
When reviewing, please take a look at the README and the project descriptions. Had to extract the integration description and not entirely sure they were all captured correctly.

File: README.md [hunk 1]
```diff
@@ -1,42 +1,19 @@
...
-[Terraform templates for additional accounts creating new CloudTrail log](https://github.com/CrowdStrike/Cloud-AWS/tree/master/Discover-for-Cloud-Templates/AWS/terraform-templates/log-archive-account)
-
-[Terraform templates for additional accounts using and existing CloudTrail log](https://github.com/CrowdStrike/Cloud-AWS/tree/master/Discover-for-Cloud-Templates/AWS/terraform-templates/additional-account-existing-trail)
-
-The python script "register_account.py" is included as an example of a script that should be run at the end of the terraform apply to register the AWS account with Crowdstrike.  The script may be run as part of a pipeline or as a local-exec process.
-
-### CloudFormation
-
-[See the README.md file here](https://github.com/CrowdStrike/Cloud-AWS/tree/master/Discover-for-Cloud-Templates/AWS/cloudformation-templates)
+![](https://raw.githubusercontent.com/CrowdStrike/falconpy/main/docs/asset/cs-logo.png)
```
Comment (shawndwells) [inline README.md:1]: Note this URL will be updated to this project after merge.

**Comment by jshcodes** [comment]
> This PR establishes the initial GitBooks scaffolding and table of contents. The project's primary README is also updated for clarity.
> 
> * PR is built against my fork and displayed at https://crowdstrike.gitbook.io/cloud-aws/. Once this PR is merged, will cut over the https://crowdstrike.gitbook.io/cloud-aws/ to this upstream repo.
> * Updated the wordlist by adding `deregister`, then sorted the whole file alphabetically.

Added "privatelink" to the wordlist due to links in the README causing the workflow to flow.

**Comment by shawndwells** [comment]
> > This PR establishes the initial GitBooks scaffolding and table of contents. The project's primary README is also updated for clarity.
> > 
> > * PR is built against my fork and displayed at https://crowdstrike.gitbook.io/cloud-aws/. Once this PR is merged, will cut over the https://crowdstrike.gitbook.io/cloud-aws/ to this upstream repo.
> > * Updated the wordlist by adding `deregister`, then sorted the whole file alphabetically.
> 
> Added "privatelink" to the wordlist due to links in the README causing the workflow to fail.

Ok, thanks!

**Comment by shawndwells** [comment]
image

**Review by redhatrises — APPROVED** [review]

**Comment by shawndwells** [comment]
https://crowdstrike.gitbook.io/cloud-aws/ updated to this repo! :)

**Pull request merged**